src/collections/resources/Resources.style.js

```diff
@@ -222,7 +222,14 @@ div.fact-left {
     border-bottom: 1px dashed ${(props) => props.theme.primaryLightColor};
   }
 
-
+  /* ==============
+  styles for hpes-adoption-of-meshery-and-meshmap
+  =======================*/
+  
+  .benefits{
+    color: ${props => props.theme.whiteToBlack};
+  }
+  
```

Comment:
Please move this to the HPE stylesheet

Reply:
@leecalcote I have made the required changes. Please review it.
Note: The div of the 4 benefit points contained inline css, I moved it to the HPE stylesheet for consistency.

Reply:
Nice!